pathcalc/TPathCalculator.cxx

```diff
@@ -120,6 +120,8 @@ bool TPathCalculator::CalculatePath(XYPoint start, XYPoint end, std::vector <XYP
 
     // Check PMT collision only if height_OpticalBox >= 0, where 0 represents top of PMT
     // and 1 represents 1cm below the top of PMT, 2 represents 2cm, etc.
+    // Temporarily disabling
```

Comment:
Just a reminder for the future: I guess this (and similarly in `TRotationCalculator.cxx`) will need to be tested later.